### PR TITLE
Prevent concatenated console lines

### PR DESCRIPTION
**Pull request by jean-m-cyr**
Create a copy of the constant parameter, append a new-line, send it with single << operator

### CODE REVIEW

File: libdevcore/Log.cpp [hunk 1]
```diff
@@ -155,5 +155,7 @@ void dev::setThreadName(char const* _n)
 
 void dev::simpleDebugOut(std::string const& _s)
 {
-	std::cerr << _s << '\n';
+        string s = _s;
```
Comment (chfast) [inline libdevcore/Log.cpp]: If you need a copy, make it by passing the argument by value.

Comment (jean-m-cyr) [reply libdevcore/Log.cpp]: Ok, but what's the difference? Still ends up with a copy.

Comment (jean-m-cyr) [reply libdevcore/Log.cpp]: Wanted the copy code invoked in one place, not with every call.

Changed

cout << a << b;

to

cout << a + b;

**Comment by jean-m-cyr** [comment]
@chfast How about this latest commit. The copy remains in the simplePrint method instead of with every call to it, and the constant attribute is retained on the parameter.

**Pull request merged**